docs/drawSeries.md

```diff
@@ -0,0 +1,35 @@
+## jquery.flot.drawSeries.js
+
+This plugin is used by flot for drawing lines, plots, bars or area.
```

Comment:
Somehow, somewhere this information has to be mentioned:

There are 2 types of plugins: optional and *part of the flot* but moved outside the main file for better testing. 

Maybe here: https://github.com/ni-kismet/engineering-flot/blob/master/PLUGINS.md